suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -321,8 +321,19 @@ const filterAnalyzeResult = (result: Analyze) => {
 export const analyzeTransactions = (
     fresh: AccountTransaction[],
     known: WalletAccountTransaction[],
-    context: { blockHeight: number },
+    context: { blockHeight: number; networkType?: Account['networkType'] },
 ) => {
+    if (context?.networkType === 'solana') {
+        const newTransactions = fresh.filter(freshTx =>
+            known.every(currentTx => currentTx.txid !== freshTx.txid),
+        );
```

Comment:
Could this perhaps be changed to:
```javascript
const knownIdsSet = new Set(known.map(tx => tx.txid));
const newTransactions = fresh.filter(freshTx => !knownIdsSet.has(freshTx.txid));
 ```
The current code might be too slow if there's a lot of transactions, no?

 cc @PeterBenc